contracts/style.md

```diff
@@ -28,10 +46,66 @@ Our starting point is the [official Solidity Style Guide](https://solidity.readt
 - Internal/private constants should be all caps with underscores: `FOO_BAR`. Like other contract variables, constants should not be public. Create getter methods if you want to publicly expose constants.
 - Explicitly declare variable size: `uint256` not just `uint`. In addition to being explicit, it matches the naming used to calculate function selectors.
 
+Examples:
+
+Good:
+
+```javascript
+uint256 private s_myVar;
+uint256 private immutable i_myImmutVar;
+uint256 private constant MY_CONST_VAR;
+
+function getMyVar() external view returns(uint256){
+    return s_myVar;
+}
+```
+
+Bad:
+
+```javascript
+uint private s_myVar;
+uint256 private immutable myImmutVar;
+uint256 private constant s_myConstVar;
+
+function getMyVar_() external view returns(uint256){
+    return s_myVar;
+}
+```
```

Comment:
Github's suggestion feature isn't working well with the markdown code blocks inside of markdown comment, so I delimited with...

I'd suggest changing the bad example to something like:
```
function getMyVar_(uint _multiplier) external view returns(uint256){
    return _mutliplier * s_myVar;
```
and updating the good version to mirror it. Reason being I think the `uint` in the function signature is especially disruptive, so possibly worth demonstrating. Used the extra parameter to also demonstrate the discouraged leading underscore on params.

Reply:
Updated! What you think?